.travis.yml

```diff
@@ -21,9 +21,9 @@ install:
 before_script:
   - "sekexe/run 'mount -t tmpfs -o size=8g tmpfs /var/lib/docker && docker -d -H tcp://0.0.0.0:2375' &"
   - "while ! docker info &> /dev/null ; do sleep 1; done"
-  - docker build -t newman-docker-ubuntu-14_04 ./ubuntu_14_04
+  - docker build -t newman-docker-ubuntu-1404 ./ubuntu_1404
```

Comment:
It would have been good to build the image in correct namespace tag so that very little needs to be changed for pushing to our repository.

`postman/newman_ubuntu1404`